PackagePrice.java

```diff
@@ -0,0 +1,56 @@
+package pricecomparison;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class PackagePrice {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/package/get-price-req?pkgId=78350&ahdate=14%2F05%2F2016";
+		driver.get(appUrl);
+
+		List<WebElement> allelements = driver

```

Comment:
Java Convention for variable names is camel case with first alphabet lower. This should be defined as allElements and not allelements.